### DISCUSSION TITLE
MQTTSubscribe.py: error

### DISCUSSION THREAD

**Discussion by hwa59** [bellrichm/WeeWX-MQTTSubscribe#217] [Q&A]
Hi
I made a fresh install of MQTTSubscribe. After Installation i tried to generate an example-configuration but when I try 
"python3 $USER_ROOT/MQTTSubscribe.py configure --create-example mqttsubscribe.template.conf" I got the following error
"MQTTSubscribe.py: error: unrecognized arguments: --create-example mqttsubscribe.template.conf"

**Comment by hwa59** [comment]
Hi
found the answer by myself. I had the wrong manual. i installed the version 2.3.2 and had the manual from verson 3.XX. With the right manual it worked.
My mistake sorry

**Reply by bellrichm** [reply]
Good to know. Thanks for the update.
